app/controllers/repairs_controller.rb

```diff
@@ -13,6 +13,7 @@ def new
 
   def create
     @repairs = Repair.new(repair_params)
+    @repairs.owner_id = current_user.owner.id
```

Comment:
There is a potential problem here in that current_user.owner may be nil, in which case this will throw an error.  You want to check that, and redirect to the owner new page if owner is nil.